src/pages/Job/index.js

```diff
@@ -17,7 +17,7 @@ import { highlight, languages } from "prismjs/components/prism-core";
 import "prismjs/components/prism-yaml";
 import "prismjs/themes/prism-tomorrow.css";
 import { Helmet } from "react-helmet";
-
+import moment from 'moment';
```

Comment:
"moment"

Reply:
Also, missing `moment` in package.json, you can install it by using `npm install moment`

Reply:
Right. Seems like I forgot to remove this import since I haven't used it the code. It would be better if I will just remove it.